Sources/Commands/SwiftPackageTool.swift

```diff
@@ -68,11 +68,61 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
 
         case .update:
             let workspace = try getActiveWorkspace()
-            try workspace.updateDependencies(
-                root: getWorkspaceRoot(),
-                diagnostics: diagnostics
-            )
+            if options.packageUpdateDryRun {
+                guard let result = try workspace.updateDependenciesDryRun(
+                    root: getWorkspaceRoot(),
+                    diagnostics: diagnostics
+                ) else {
+                        return
+                }
+                
+                var maxWidth = 0
+                for (package, state) in result {
+                    if state != .unchanged {
+                        if let packageName = package.name {
+                            let packageString = "\(packageName)"
+                            let stateString = "\(state)"
```

Comment:
We shouldn't use builtin description for the state. We can add a method that converts the value of state that is more suitable for users to read.